test/unit/services/domain/test-workload-points.js

```diff
@@ -81,6 +81,55 @@ describe('services/domain/workload-points', function () {
     expect(workloadPoints.isT2A).to.equal('false')
   })
 
+  it('should create a valid T2A workload points object', function () {
+    var t2aWorkloadPoints = new WorkloadPoints(Object.assign({}, VALID_WORKLOAD_POINTS))
+    t2aWorkloadPoints.isT2A = 'true'
+    t2aWorkloadPoints.defaultContractedHoursPo = '0'
```

Comment:
If you're assigning the values here and checking the below, it's not really testing the `WorkloadPoints` function. I'm not sure this test is needed

Reply:
I agree, removed the test.